content/components/only-manipulate-the-dom-via-the-renderer.md

```diff
@@ -0,0 +1,74 @@
+---
+title: only manipulate the DOM via the Renderer
+author: 
+  name: Billy Lando
+  url: https://github.com/billyjov
+---
+
+# Problem 
+
+According to the Angular documentation, relying on direct DOM access creates tight coupling between your application and rendering layers which will make it impossible to separate the two and deploy your application into a web worker. 
+
+Consequently, using jQuery , `document` object, or `ElementRef.nativeElement` is not recommended as it's not available on other platforms such as server-side rendering or web worker.
```

Comment:
I still think that `server-side rendering` is a little bit off in this sentence because we talk about platforms and SSR in particular is not a platform.

I'd go with:

> Consequently, using jQuery , `document` object, or `ElementRef.nativeElement` is not recommended as it's not available on other platforms such as server (for server-side rendering) or web worker.

Reply:
@billyjov Any updates on this one? I'd really love to get this merged.

Reply:
@d3lm i'm on vacation and will take a look at this next week.

Reply:
Got it! Thanks.

Reply:
done